platforms/native/engine/jsb-loader.js

```diff
@@ -422,21 +425,22 @@ parser.register({
 
     '.ExportJson': parseJson,
 });
-
-cc.assetManager.transformPipeline.append((task) => {
-    const input = task.output = task.input;
-    for (let i = 0, l = input.length; i < l; i++) {
-        const item = input[i];
-        if (item.config) {
-            item.options.__cacheBundleRoot__ = item.config.name;
-        }
-        if (item.ext === '.cconb') {
-            item.url = item.url.replace(item.ext, '.bin');
-        } else if (item.ext === '.ccon') {
-            item.url = item.url.replace(item.ext, '.json');
+if (CC_BUILD) {
+    cc.assetManager.transformPipeline.append((task) => {
+        const input = task.output = task.input;
+        for (let i = 0, l = input.length; i < l; i++) {
+            const item = input[i];
+            if (item.config) {
+                item.options.__cacheBundleRoot__ = item.config.name;
+            }
+            if (item.ext === '.cconb') {
```

Comment:
Can not transform `cconb` to `bin` in preview and editor since there is only cconb in editor.

Reply:
Under what circumstances CC_BUILD exists？

Reply:
CC_BUILD = !CC_EDITOR && !CC_PREVIEW && !CC_DEV